static/js/common/init.js

```diff
@@ -24,24 +24,63 @@ MEDUSA.common.init = function() {
         return offset;
     }
 
+    let initHorizontalScroll = function() {
```

Comment:
It's to check if we need to show the left and right buttons. It tries to do that using the scrollWidth, clientWidth values. I'm running it on init and on resize